execution/executor/src/components/chunk_output.rs

```diff
@@ -76,7 +76,7 @@ impl ChunkOutput {
             maybe_block_gas_limit,
         )?;
 
-        // TODO(skedia) add logic to emit counters per shard instead of doing it globally.
+        update_counters_for_processed_chunk(&transactions, &transaction_outputs, "executed");
```

Comment:
Why did this change? We don't have a good way to support these counters in sharded execution for now.